install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,30 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes only receive labels if you explicitly add them in the inventory using
+`*openshift_node_labels*` as described below. `*region=infra*` is the default
+selector. If you create nodes with that selector, then the router and registry
+will be installed.
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
```

Comment:
@ahardin-rh Sorry, i should have picked up on this yesterday, but is the "default selectors for router and registry services" statement supposed to be a uncommented meant for context? The other two lines are commands for the reader to run, right? If this is all true, the I think that line can probably be removed. The para above is stating that anyway.